torch_em/data/datasets/ctc.py

```diff
@@ -88,7 +100,8 @@ def get_ctc_segmentation_dataset(
     path,
     dataset_name,
     patch_shape,
-    split=None,
+    split,
```

Comment:
Let's set the default value for `split` to `"train"`, since the test split does currently not work.

Reply:
Done!